.github/CODEOWNERS

```diff
@@ -1,5 +1,26 @@
 # Code owners file.
 # This file controls who is tagged for review for any given pull request.
+#
+# What is a "CODEOWNER"?
+#
+# A CODEOWNER MUST ____ for packages they own:
+# - introduce themselves at a Python SIG meeting
+# - demonstrate advanced knowledge of the dependencies
```

Comment:
```suggestion
# - have enough knowledge of the corresponding instrumented library
```

Considering the fact that we probably won't have advanced knowledge of the instrumented library, it won't make much sense that we ask others to demonstrate theirs. :shrug: